.devcontainer/Dockerfile

```diff
@@ -0,0 +1,14 @@
+# [Choice] Node.js version (use -bullseye variants on local arm64/Apple Silicon): 18, 16, 14, 18-bullseye, 16-bullseye, 14-bullseye, 18-buster, 16-buster, 14-buster
+ARG VARIANT=16-bullseye
+FROM mcr.microsoft.com/vscode/devcontainers/javascript-node:${VARIANT}
+
+# [Optional] Uncomment this section to install additional OS packages.
```

Comment:
Should we remove this commented out config ?

Reply:
I would like to put them here because I think we will need to install other packages in the future and this is a nice convention to reference  🤔 Is it good for you?

Reply:
when the future comes and we hit problem, we will add it.
Trying to make everything as clean as possible